libs/db/debug_db.go

```diff
@@ -1,257 +0,0 @@
-package db
-
-import (
-	"fmt"
-	"sync"
-
-	cmn "github.com/tendermint/tendermint/libs/common"
-)
-
-//----------------------------------------
-// debugDB
-
-type debugDB struct {
```

Comment:
this can be used for debugging. I've never used it myself, but maybe Cosmos-SDK devs had & need it.

Reply:
its set as `const = false` in the code in the sdk. cc @alexanderbez @alessio @jackzampolin @rigelrozanski have you guys used newdbDebug at anytime